server/metrics/fdb.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022 Tigris Data, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"github.com/uber-go/tally"
+	"strconv"
+)
+
+var (
+	FdbErrorRequests    tally.Scope
```

Comment:
Unexport, if it won't be used outside of this package.

Reply:
This is used in kv, measure() increases these counters.